build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1679,6 +1679,9 @@ message Compressor {
 
     // Zstandard compression.
     ZSTD = 1;
+
+    // RFC 1951 Deflate, commonly used by ZIP files.
```

Comment:
Since there are a few interrelated forms of deflate, I think this should be more explicit- IIUC you mean "raw deflate", correct? Is it worth saying "without zlib or gzip structure" and dropping the reference to zip?

Reply:
Isn't "RFC 1951 Deflate" already unambiguously describing what the format is?

Reply:
Technically yes, but we've had a couple of decades of `Content-Encoding: deflate` meaning something other than "plain deflate" to confuse things. Maybe it's just me though.

Reply:
I've extended the description. At the same time, I've made it more explicit that this algorithm is not the recommended one.